presto-hive/src/main/java/com/facebook/presto/hive/rule/HiveFilterPushdown.java

```diff
@@ -376,7 +376,7 @@ public PlanNode visitFilter(FilterNode filter, Void context)
             RowExpression replacedExpression = replaceExpression(expression, symbolToColumnMapping);
             // replaceExpression() may further optimize the expression; if the resulting expression is always false, then return empty Values node
             if (FALSE_CONSTANT.equals(replacedExpression)) {
-                return new ValuesNode(tableScan.getSourceLocation(), idAllocator.getNextId(), tableScan.getOutputVariables(), ImmutableList.of());
+                return new ValuesNode(tableScan.getSourceLocation(), idAllocator.getNextId(), tableScan.getOutputVariables(), ImmutableList.of(), Optional.of(tableScan.getTable().toString()));
```

Comment:
`getTable()` -> `getTable().getConnectorHandle()`. Same for other places in the file

Reply:
Done